test/fixtures/command-line/help-test.txt

```diff
@@ -88,7 +88,7 @@ Options:
                                       specifying a custom connectivity profile)
   -U, --bwup <bandwidth>              upload bandwidth in Kbps (used when
                                       specifying a custom connectivity profile)
-  -thc, --cputhrottle <number>        Custom cpu throttling                                   
+  -thc, --throttleCPU <number>        Custom cpu throttling                                   
```

Comment:
Need to make this "custom cpu throttling" for the test to pass (note the lowercase)